scripts/common.py

```diff
@@ -116,6 +116,9 @@
 CONSTANTS_FILE_PATH = os.path.join('assets', 'constants.ts')
 MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS = 1000
 REDIS_CONF_PATH = os.path.join('.', 'redis.conf')
+# Path for the dump file the redis server autogenerates. It contains data
+# used by the Redis server.
+REDIS_DUMP_PATH = os.path.join(CURR_DIR, 'dump.rdb')
```

Comment:
Should we also add 'dump.rdb' to the `.gitignore`? [it'll help developers to avoid this file as untracked file in the local repo.]

Reply:
Already added in the original redis PR.

Reply:
Thanks for the info, I had seen this file in the untraced files list once, I don't know how and that's why I asked! :)